lib/config/detekt/baseline.xml

```diff
@@ -2,6 +2,8 @@
 <SmellBaseline>
   <ManuallySuppressedIssues></ManuallySuppressedIssues>
   <CurrentIssues>
-    <ID>FunctionOnlyReturningConstant:Library.kt$Library$fun someLibraryMethod(): Boolean</ID>
+    <ID>LongParameterList:EntityService.kt$EntityService$( brokerUrl: String, authServerUrl: String, authClientId: String, authClientSecret: String, authGrantType: String, queryParams: Map&lt;String, String&gt;, contextUrl: String )</ID>
+    <ID>LongParameterList:SubscriptionService.kt$SubscriptionService$( brokerUrl: String, authServerUrl: String, authClientId: String, authClientSecret: String, authGrantType: String, subscriptionPayload: String )</ID>
+    <ID>SwallowedException:UriUtils.kt$catch (e: URISyntaxException) { null }</ID>
```

Comment:
what is this?

Reply:
It's added in recent versions of Detekt, i'm trying to find how to fix it.
From detekt: `when exceptions are caught and not correctly passed (e.g. as a cause) into a newly thrown exception.  `
I tried to throw a new unparseableException (as done in stellio with a BadRequestDataException) but it stills in the baseline.

Reply:
@bobeal, I updated the PR, but still have this Exception. 
I added a log when the exception occurs as mentioned [here](https://detekt.github.io/detekt/exceptions.html#compliant-code-4).

Reply:
I fixed it